sling/core/pckgmgr/src/main/java/com/composum/sling/core/pckgmgr/PackageServlet.java

```diff
@@ -139,7 +139,7 @@ public void init() throws ServletException {
         operations.setOperation(ServletOperationSet.Method.POST, Extension.json,
                 Operation.install, new InstallOperation());
         operations.setOperation(ServletOperationSet.Method.POST, Extension.json,
-                Operation.deploy, new DeployOperation());
+                Operation.deploy, new CrxServiceOperation());
```

Comment:
CRX (aka Content Repository eXtreme ) is a set of commercial extensions to Apache Jackrabbit ( 2.x ) which is part of AEM.